src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/PhysicalMarker/ArUcoThreeMarkerSpatialLocalizer.cs

```diff
@@ -26,7 +26,7 @@ protected override bool IsSupported
         {
             get
             {
-#if UNITY_EDITOR
+#if UNITY_EDITOR || UNITY_WSA
```

Comment:
> || UNITY_WSA [](start = 16, length = 13)

If you're going to change this, please use the pattern that ArUcoMarkerSpatialLocalizer uses so that it's only supported on x86 UWP platform:

#if UNITY_EDITOR
                return true;
#elif UNITY_WSA
                return Windows.ApplicationModel.Package.Current.Id.Architecture == Windows.System.ProcessorArchitecture.X86;
#else
                return false;
#endif